script.js

```diff
@@ -0,0 +1,115 @@
+const opBtn = document.getElementById('open-Button');
+const cBtn = document.getElementById('close-Button');
+const sNav = document.getElementById('sNav');
+
+opBtn.addEventListener('click', () => {
+  sNav.style.width = '100%';
+});
+cBtn.addEventListener('click', () => {
+  sNav.style.width = '0';
+});
+
+const spData = [
+  {
+    id: 1,
+    Name: 'Grey Nguyen',
+    Title: 'Senior Software Developer with more than 15 years experience', 
+    Desc: 'Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006',
+    image: './images/speaker_2.png',
+  },
+  {
+    id: 2,
+    Name: 'Lila tretikov', 
+    Title: 'Executive Director of the Wikipedia Foundation', 
+    Desc: 'Lila Tretikov is the Executive of the Wikimedia Foundation, the nonprofit organization that operates Wikipedia. Wikipedia is freely available in 290 languag-es and used by nearly half a billion people around the world every month.',
```

Comment:
Please pay attention to detail your first two speaker title section has a black font colour in your mobile view please follow the colour in the template design.

This is your design for the first two speakers:
![Screenshot 2023-03-15 at 6 00 09 PM](https://user-images.githubusercontent.com/79506105/225370626-7c50e6c3-65a9-4d99-a439-80143240f868.png)


Please change the title font in your first two speakers to the orange font as is in the original template design.